libs/client/use-cases/dashboard/shared/ui/src/lib/footer/footer.component.scss

```diff
@@ -0,0 +1,32 @@
+.footer {
+    font-family: Arial, sans-serif;
+    background: #317874;
+    height: 6em;
+    bottom: 0;
+    width: 100%;
+    position: fixed;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+    padding: 1rem;
+    font-size: medium;
+    // font-stretch: extra-condensed;
+}
+
+.footer__item {
+    font-family: Arial, sans-serif;
```

Comment:
I suggest not to use another font family here, just have 1 font to start with, it can be set globally.
for now you can just delete this font family